sdk/src/Core/Amazon.Runtime/Telemetry/Metrics/AsyncMeasurement.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+namespace Amazon.Runtime.Telemetry.Metrics
+{
+    /// <summary>
+    /// Represents an async measurement.
+    /// An async measurement records values, such as periodically or based on events.
+    /// The generic type parameter T specifies the type of value being recorded, such as int, long, etc.
```

Comment:
Did you intend these lines to show up with newlines in intelliSense? They won't without <para>. This same comment applies to other /// comments in this review

Reply:
I think it looks fine/better without the `<para>` tags. I was only mentioning if you were trying to format your comment which it looks like you were that it wouldn't show up that way in intelliSense.